aten/src/ATen/native/UnaryOps.cpp

```diff
@@ -31,6 +31,101 @@
 namespace at {
 namespace native {
 
+// Using anonymous namespace as these functions' scope is limited to this file and we do not want to pollute at::native namespace
```

Comment:
don't need a comment, it's a fairly common pattern across all of pytorch

Reply:
Okay! Removed the comment :)